projects/cashmere/src/lib/toaster/hc-toast-ref.ts

```diff
@@ -2,8 +2,9 @@ import {OverlayRef} from '@angular/cdk/overlay';
 import {HcToastComponent} from './hc-toast.component';
 import {Subject} from 'rxjs';
 import {filter, take} from 'rxjs/operators';
-import {Input} from '@angular/core';
+import { Input, Directive } from '@angular/core';
 
+@Directive()
```

Comment:
Is this really a Directive?  Seems like it should instead just be an `@Injectable()`

Reply:
Wow, great catch.  This has been in the library for ages and never got noticed, but you're absolutely right.  Change made.